bin/interpreter.sh

```diff
@@ -84,7 +84,6 @@ if [[ -d "${ZEPPELIN_HOME}/zeppelin-zengine/target/test-classes" ]]; then
 fi
 
 addJarInDirForIntp "${ZEPPELIN_HOME}/zeppelin-interpreter-api/target"
-addJarInDirForIntp "${ZEPPELIN_HOME}/lib/interpreter"
```

Comment:
`addJarInDirForIntp "${ZEPPELIN_HOME}/lib/interpreter"` cannot be deleted,
I tested it and if I delete this line, I can't start the interpreter.
Without deleting this line of code, the interpreter can be started normally.

Or, do you test it?

Reply:
I copy all the interpreters into `lib` instead of `lib/interpreter`, I just tested locally and it works. Could you try this ?

Reply:
Ok, I didn't notice this.